<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WatchWindow</class>
 <widget class="QMainWindow" name="WatchWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WatchWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="INFO">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Processing</string>
       </attribute>
       <widget class="QTextEdit" name="textEdit">
        <property name="geometry">
         <rect>
          <x>610</x>
          <y>10</y>
          <width>161</width>
          <height>41</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="FIND">
        <property name="geometry">
         <rect>
          <x>670</x>
          <y>80</y>
          <width>89</width>
          <height>25</height>
         </rect>
        </property>
        <property name="text">
         <string>FIND</string>
        </property>
       </widget>
       <widget class="QPushButton" name="KILL">
        <property name="geometry">
         <rect>
          <x>670</x>
          <y>120</y>
          <width>89</width>
          <height>25</height>
         </rect>
        </property>
        <property name="text">
         <string>KILL</string>
        </property>
       </widget>
       <widget class="QPushButton" name="renew">
        <property name="geometry">
         <rect>
          <x>680</x>
          <y>350</y>
          <width>89</width>
          <height>25</height>
         </rect>
        </property>
        <property name="text">
         <string>RENEW</string>
        </property>
       </widget>
       <widget class="QTableView" name="tableView">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>581</width>
          <height>371</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="next">
        <property name="geometry">
         <rect>
          <x>610</x>
          <y>80</y>
          <width>51</width>
          <height>25</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="text">
         <string>next</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Resources</string>
       </attribute>
       <widget class="QWidget" name="CPU" native="true">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>611</width>
          <height>171</height>
         </rect>
        </property>
       </widget>
       <widget class="QProgressBar" name="Mem">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>300</y>
          <width>271</width>
          <height>23</height>
         </rect>
        </property>
        <property name="value">
         <number>24</number>
        </property>
       </widget>
       <widget class="QProgressBar" name="SWAP">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>350</y>
          <width>271</width>
          <height>23</height>
         </rect>
        </property>
        <property name="value">
         <number>24</number>
        </property>
       </widget>
       <widget class="QLabel" name="CPUTitle">
        <property name="geometry">
         <rect>
          <x>660</x>
          <y>100</y>
          <width>91</width>
          <height>17</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>13</pointsize>
         </font>
        </property>
        <property name="text">
         <string>CPU_ RATE</string>
        </property>
       </widget>
       <widget class="QLabel" name="MEM_RATE">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>300</y>
          <width>91</width>
          <height>17</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>13</pointsize>
         </font>
        </property>
        <property name="text">
         <string>MEM RATE</string>
        </property>
       </widget>
       <widget class="QLabel" name="SWAP_RATE">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>350</y>
          <width>101</width>
          <height>17</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>13</pointsize>
         </font>
        </property>
        <property name="text">
         <string>SWAP RATE</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>About</string>
       </attribute>
       <widget class="QGroupBox" name="groupBox">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>501</width>
          <height>351</height>
         </rect>
        </property>
        <property name="title">
         <string>INFOMATION</string>
        </property>
        <widget class="QLabel" name="label">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>40</y>
           <width>401</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_2">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>90</y>
           <width>421</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_3">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>140</y>
           <width>421</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_4">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>190</y>
           <width>411</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_5">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>300</y>
           <width>451</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </widget>
       <widget class="QPushButton" name="pushButton">
        <property name="geometry">
         <rect>
          <x>600</x>
          <y>340</y>
          <width>141</width>
          <height>25</height>
         </rect>
        </property>
        <property name="text">
         <string>ShutDown SysTem</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
